eldritch/items.py

```diff
@@ -238,8 +239,41 @@ def DreadCurse(idx):
   return CombatSpell("Dread Curse", idx, {"magical": 9}, 2, -2, 2)
 
 
-class EnchantWeapon(CombatSpell):
+class BindMonster(CombatSpell):
+  def __init__(self, idx):
+    super().__init__(
+        "Bind Monster", idx, {"magical": float("inf"), "physical": float("inf")}, 2,
+        4, 2,
+    )
+    self.combat_round = None
+
+  def get_required_successes(self, state):  # pylint: disable=unused-argument
+    self.combat_round = state.event_stack[
+        -3
+    ]  # CombatRound[-3] > CombatChoice[-2] > CastSpell[-1]
+    return self.combat_round.monster.toughness(state, self.combat_round.character)
+
+  def get_usable_interrupt(self, event, owner, state):
+    if isinstance(event, events.CombatChoice) and isinstance(
```

Comment:
Might just update the CombatChoice constructor to always take a monster as an argument (even if it's None) to make sure that the monster is always set on the CombatChoice. I'm honestly not sure why I didn't do that before.